vault/token_store.go

```diff
@@ -3183,6 +3241,13 @@ func (ts *TokenStore) tokenStoreRoleCreateUpdate(ctx context.Context, req *logic
 		}
 	}
 
+	allowedEntityAliasesRaw, ok := data.GetOk("allowed_entity_aliases")
+	if ok {
+		entry.AllowedEntityAliases = strutil.RemoveDuplicates(allowedEntityAliasesRaw.([]string), true)
+	} else if req.Operation == logical.CreateOperation {
```

Comment:
I think you don't need this else block -- we'll never have a default that isn't nil.

Reply:
Makes sense. Thanks.